src/lib.rs

```diff
@@ -438,6 +509,11 @@ where
     {
         self.storage.purge();
     }
+
+    #[allow(dead_code)] // FIXME
```

Comment:
unaddressed fixme

Reply:
Bah. Thanks.

Reply:
And indeed the function appears to be dead. lol.